demisto_sdk/commands/init/initiator.py

```diff
@@ -99,6 +108,74 @@ def init(self):
+                metadata_dict['author'] = metadata.get('author', '')
+                metadata_dict['support'] = metadata.get('support', '')
+                metadata_dict['url'] = metadata.get('supportDetails', {}).get('url', '')
+                metadata_dict['email'] = metadata.get('supportDetails', {}).get('email', '')
+        if os.path.exists(os.path.join(packs_dir, pack_name)):
+            print_color(
+                f'Modifying pack name because pack {pack_name} already exists in the content repo', LOG_COLORS.NATIVE
+            )
+            if pack_name[-2].lower() == 'v' and pack_name[-1].isdigit():
+                # increment by one
+                pack_name = pack_name[:-1] + str(int(pack_name[-1]) + 1)
+            else:
+                pack_name += 'V2'
+            print_color(f'New pack name is "{pack_name}"', LOG_COLORS.NATIVE)
+        pack_dir = os.path.join(packs_dir, pack_name)
+        os.mkdir(pack_dir)
+        shutil.unpack_archive(filename=self.contribution, extract_dir=pack_dir)
+        pack_subdirectories = get_child_directories(pack_dir)
+        for pack_subdir in pack_subdirectories:
+            basename = os.path.basename(pack_subdir)
+            if basename in ENTITY_TYPE_TO_DIR:
+                dst_name = ENTITY_TYPE_TO_DIR.get(basename)
+                src_path = os.path.join(pack_dir, basename)
+                dst_path = os.path.join(pack_dir, dst_name)
+                content_item_dir = shutil.move(src_path, dst_path)
+                if basename in {'integration', 'automation'}:
+                    self.content_item_to_package_format(content_item_dir, del_unified=True)
+        # create pack's base files
+        self.full_output_path = pack_dir
+        self.create_pack_base_files()
+        metadata_dict = Initiator.create_metadata(fill_manually=False, data=metadata_dict)
+        metadata_path = os.path.join(self.full_output_path, 'pack_metadata.json')
+        with open(metadata_path, 'w') as pack_metadata_file:
+            json.dump(metadata_dict, pack_metadata_file, indent=4)
+        # remove metadata.json file
+        os.remove(os.path.join(pack_dir, 'metadata.json'))
+
+    def content_item_to_package_format(self, content_item_dir, del_unified):
+        '''
+        Iterate over the YAML files in a directory and create packages (a containing directory and
+        component files) from the YAMLs of integrations and scripts
+
+        Args:
+            content_item_dir (str): Path to the directory containing the content item YAML file(s)
+            del_unified (bool): Whether to delete the unified yaml the package was extracted from
+        '''
+        child_files = get_child_files(content_item_dir)
+        content_item_file_path = ''
+        for child_file in child_files:
+            cf_name_lower = os.path.basename(child_file).lower()
+            if cf_name_lower.startswith(('integration', 'automation')) and cf_name_lower.endswith('yml'):
```

Comment:
lets use the constants we have for these entities
lets also search for `script` in case we add it in the future